src/multipart_.rs

```diff
@@ -205,6 +226,7 @@ impl fmt::Debug for Part {
             .field("value", &self.value)
             .field("mime", &self.mime)
             .field("file_name", &self.file_name)
+            .field("hdr", &self.hdr)
```

Comment:
For debugging, we may as well call it the full word, `headers`.

Reply:
I renamed the struct field, too…